src/gtrack.cxx

```diff
@@ -79,6 +82,7 @@ GTrack::GTrack(int x, int y, int w, int h, const char* l ) :
 
 	jackSendActivate.setColor( 1, 1, 0 );
 	jackSendActivate.callback(gtrack_jacksendactivate_cb,this);
+	jackSendActivate.tooltip("Add FX");
```

Comment:
"Add FX" sounds like it could be a Luppp internal FX unit? Perhaps "Activate JACK FX send" is a better description?